stable/stackdriver-exporter/README.md

```diff
@@ -0,0 +1,103 @@
+# Stackdriver Exporter
+
+Prometheus exporter for Stackdriver, allowing for Google Cloud metrics.  You
+must have appropriate IAM permissions for this to exporter to work.  If you
+are passing in an IAM key then you must have:
+
+* monitoring.metricDescriptors.list
+* monitoring.timeSeries.list
+
+These are contained within `roles/monitoring.viewer`.  If you're using legacy
+access scopes, then you must have
+`https://www.googleapis.com/auth/monitoring.read`.
+
+Learn more: https://github.com/justwatchcom/stackdriver_exporter 
+
+## TL;DR;
+
+```bash
+$ helm install stable/stackdriver-exporter --set stackdriver.project_id=google-project-name
+```
+
+## Introduction
+
+This chart creates a Stackdriver-Exporter deployment on a
+[Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh)
+package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+
+## Installing the Chart
+
+To install the chart with the release name `my-release`:
+
+```bash
+$ helm install --name my-release stable/stackdriver-exporter --set stackdriver.project_id=google-project-name
+```
+
+The command deploys Stackdriver-Exporter on the Kubernetes cluster using the
+default configuration. The [configuration](#configuration) section lists the
+parameters that can be configured during installation.
+
+## Uninstalling the Chart
+
+To uninstall/delete the `my-release` deployment:
+
+```bash
+$ helm delete --purge my-release
+```
+The command removes all the Kubernetes components associated with the chart and
+deletes the release.
+
+## Configuration
+
+The following table lists the configurable parameters of the
+Stackdriver-Exporter chart and their default values.
+
+Parameter                           | Description                          | Default
+----------------------------------- | ------------------------------------ | --------------------------------
+`replicaCount`                      | desired number of pods               | `1`
+`restartPolicy`                     | container restart policy             | `Always`
+`image.repository`                  | container image repository           | `justwatch/stackdriver_exporter`
+`image.tag`                         | container image tag                  | `1.0.2`
+`image.pullPolicy`                  | container image pull policy          | `IfNotPresent`
+`resources`                         | resource requests & limits           | `{}`
+`service.type`                      | type of service to create            | `ClusterIP`
+`service.httpPort`                  | port for the http service            | `9255`
+`stackdriver.project_id`            | GCP Project ID                       | ``
```

Comment:
Since these are YAML value keys, let's change from snake_case to lowerCamelCase (`projectID`). PS, I realize this comes from https://github.com/frodenas/stackdriver_exporter but for charts consistency sake 😄